configuration/conf/config.properties

```diff
@@ -231,6 +231,8 @@ READ_OPERATION_TIMEOUT_MS=300000
 ################## 操作信息：写入参数 ###################
 # 批写入数据行数，每行是某个设备所有传感器在某一时间戳的数据，每个Batch写入数据点数=SENSOR_NUMBER * BATCH_SIZE_PER_WRITE
 BATCH_SIZE_PER_WRITE=100
+# 每批写入设备数，必须能整除批写入数据行数
+DEVICE_NUM_PER_WRITE=10
```

Comment:
I think this parameter's default value should be 1.

Reply:
set to 1